packages/webpack/test/e2e-nodejs.spec.js

```diff
@@ -13,7 +13,7 @@ test.before(async (t) => {
     policyLocation: path.resolve(__dirname, 'fixtures/main/policy-node'),
   })
   webpackConfig.target = 'node'
-  webpackConfig.mode = 'development'
+  webpackConfig.optimization.minimize = false
```

Comment:
why disable minification on node app bundles btw?

i note removing `webpackConfig.mode = 'development'` means `production` by default (so minimizing, tree shakes, etc)

Reply:
this particular test needs to be in production mode to avoid webpack-generated module names coinciding with builtins that they're wrapping. I disable minification so that it's easier to look into the output. In production mode the module names are replaced with numbers and the old builtins implementation was naively relying on webpack using the right names which is not a great assumption.